src/components/organisms/WorldForm/components/WorldNav/WorldNavTab.ts

```diff
@@ -0,0 +1,5 @@
+export enum WorldNavTab {
```

Comment:
Maybe move this to `src/types`?

Reply:
It's just navigation specific. Unlike `AdminVenueTab` this is just moved to a separate file to reduce clutter.

Reply:
I understand, but we should keep it consistent and decide where to put them. Some of them are on component level, the rest are in `src/types`

Reply:
What is in `src/types`? are there any types there like what tab should be displayed on one component or are there types that deal with the actual customer data?

Reply:
I think it's just all types. Doesn't really matter the usage itself. If it's `interface` `type` or `enum` it's there, as far as I know.

Reply:
Based on https://github.com/sparkletown/sparkle/pull/2400#discussion_r722141060 <- not all types (wasn't distinguishing interface alias or enumeration, was distinguishing functionality).

Either way, I got nothing more to add.

Reply:
`AdminVenueTab` shouldn't have been in that file and it was missed by me and the PR review. I'd gladly move them to `src/types` the next time I touch that file. Anyways. I'll approve this to unblock it. But I will also do the same for this file the next time I have to work on this code, just to follow our latest code style.